libraries/AP_Scripting/applets/rev_thr_on_land.lua

```diff
@@ -0,0 +1,151 @@
+--[[
+    This script automatically engages reverse thrust on planes in FBWA mode on touchdown.
+    Prerequisities:
+    - ArduPlane 4.1 or later
```

Comment:
You could use the firmware version bindings to check that were actually running on plane.

Reply:
done